### PR TITLE
Updating Kubernetes network policy

### PR DESCRIPTION
**Pull request by nvunnamm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,9 @@ If you are just looking for quickly spin up an instance of RH-SSO
 # Create Network Security Policy
 oc -n 6d70e7-tools process -f openshift/nsp.yaml -p 'NAMESPACE=6d70e7-tools' | oc -n 6d70e7-tools create -f -
```
Comment (patricksimonian) [inline README.md:48]: we probably want to remove or atleast highlight you WILL not need to run aporeto policies.

File: openshift/knp.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  - description: The namespace
+    displayName: Namespace
+    name: NAMESPACE
+    required: true
+    value: sso-tools
+  - description: The environment
+    displayName: environment
+    name: ENVIRONMENT
+    required: true
+    value: ""
```
Comment (patricksimonian) [inline openshift/knp.yaml:78]: new line

**Review by patricksimonian — CHANGES_REQUESTED** [review]

**Review by patricksimonian — APPROVED** [review]
have these changes been tested in sandbox with zero aporeto based policies?

**Comment by nvunnamm** [comment]
> have these changes been tested in sandbox with zero aporeto based policies?

**Comment by nvunnamm** [comment]
closed this unexpectedly.. Reopening it.
The changes which I made are tested in sandbox.. but we had to add 2 aporeto based policies as per the guide.

**Pull request merged**